packages/postgres/package.json

```diff
@@ -39,13 +39,13 @@
   "dependencies": {
```

Comment:
https://github.com/sequelize/sequelize/blob/356239e87cf5516ca5d59eb11aea71a4ed4c3b55/packages/postgres/src/connection-manager.ts#L17

`pg-types` is not listed as a dependency anywhere, this should be added to the Postgres package.
Otherwise, looks good.

Reply:
Done